run-script-plugin-orca/src/main/kotlin/com/armory/plugin/preconfigured/RunScriptPlugin.kt

```diff
@@ -22,22 +20,11 @@ class RunScriptPlugin(wrapper: PluginWrapper): Plugin(wrapper) {
 }
 
 @Extension
-class RunScriptPreConfiguredStage(val configuration: PluginConfig) : PreconfiguredJobConfigurationProvider {
-    private val mapper: ObjectMapper = ObjectMapper(YAMLFactory()).disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-
+class RunScriptPreConfiguredStage(val pluginSdks: PluginSdks, val configuration: PluginConfig) : PreconfiguredJobConfigurationProvider {
     override fun getJobConfigurations(): List<KubernetesPreconfiguredJobProperties> {
-        val jobProperties = loadResource("com/armory/plugin/preconfigured/armory-run-script.yaml", KubernetesPreconfiguredJobProperties::class.java)
+        val jobProperties = pluginSdks.yamlResourceLoader().loadResource("com/armory/plugin/preconfigured/armory-run-script.yaml", KubernetesPreconfiguredJobProperties::class.java)
```

Comment:
I rollback the pluginSdk workaround because I remembered that this issue was fix on 1.23, I can remove the rollback if isn't true.

@danielpeach do you know if this was fixed on the 1.23 release?

Reply:
@edgarulg the plugin sdk was fixed in the 1.23 release 👍